packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -386,24 +389,37 @@ class ScheduleManagerCore {
             assert(sequenceNumber < this.pauseSequenceNumber, "we should never start processing incomplete batch!");
             // If the next op is the start of incomplete batch, then we can't process it until it's fully in - pause!
             if (sequenceNumber + 1 === this.pauseSequenceNumber) {
-                this.setPaused(true);
+                this.setPaused();
             }
         }
     }
 
-    private setPaused(localPaused: boolean) {
+    private setPaused() {
```

Comment:
Can we call these `pauseQueue` and `resumeQueue` or something like that. setPaused / setResumed are weird :).